tests/Feature/Service/ServiceWorkersPageTest.php

```diff
@@ -5,29 +5,73 @@
 use Tests\StoreTestCase;
 use Illuminate\Foundation\Testing\DatabaseMigrations;
 use App\AdminUser;
-
+use App\Centre;
+use App\CentreUser;
+use Illuminate\Support\Collection;
 
 class ServiceWorkersPageTest extends StoreTestCase
 {
     use DatabaseMigrations;
 
     /** @var AdminUser $adminUser */
     private $adminUser;
+
+    /** @var CentreUser $userHomeCentreNoAlt */
+    private $userHomeCentreNoAlt;
+
+    /** @var CentreUser $userHomeCentreTwoAlt */
+    private $userHomeCentreTwoAlt;
+
+    /** @var CentreUser $downlaoderUser */
+    private $downloaderUser;
+
+    /** @var Centre $centre */
+    private $centre;
+
+    /** @var Collection $altCentres */
+    private $altCentres;
+
     private $workersRoute;
 
     public function setUp()
     {
         parent::setUp();
+
         $this->adminUser = factory(AdminUser::class)->create();
+        // Create 3 centres in an area
+        $this->centre = factory(Centre::class, 3)->create();
```

Comment:
Need a sponsor, then bind each centre to the sponsor,

you can use `->each(function($c) {... }); `